code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -17,13 +17,13 @@
 
 /datum/ritual/cruciform/base/relief
 	name = "Relief"
-	phrase = "Et si ambulavero in medio umbrae mortis non timebo mala"
+	phrase = "Semper invicta"
 	desc = "Short litany to relieve pain of the afflicted."
 	power = 20
 	ignore_stuttering = TRUE
 
 /datum/ritual/cruciform/base/relief/perform(mob/living/carbon/human/H, obj/item/implant/core_implant/C)
-	H.add_chemical_effect(CE_PAINKILLER, 10)
+	H.add_chemical_effect(CE_PAINKILLER, 50)
```

Comment:
This doesn't actually work as intended.

Chem effects are all removed on life tick, so this actually only lasts for about 2 seconds. We should do something about this, probably.

Reply:
Is there anything I can do in this pr?

Reply:
Changed it to add to the blood stream as suggested